frontend/src/components/core/Countdown/Countdown.tsx

```diff
@@ -0,0 +1,72 @@
+/**
+ * @license
+ * Copyright 2018-2020 Streamlit Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { PureComponent, ReactNode } from "react"
+
+import "./Countdown.scss"
+
+interface Props {
+  countdown: number
```

Comment:
Why is this a prop? We always count from 3 so this should just be hard-coded.

Reply:
Because we could use this countdown in a future for a different purpose.